Resources/config/messenger.xml

```diff
@@ -11,5 +11,12 @@
         <service id="messenger.middleware.doctrine_transaction" class="Symfony\Bridge\Doctrine\Messenger\DoctrineTransactionMiddleware" abstract="true" public="false">
             <argument type="service" id="doctrine" />
         </service>
+
+        <service id="messenger.transport.doctrine.factory" class="Symfony\Component\Messenger\Transport\Doctrine\DoctrineTransportFactory">
+            <argument type="service" id="doctrine" />
+            <argument type="service" id="messenger.transport.serializer" on-invalid="ignore" />
```

Comment:
This needs to be removed now.

Reply:
Indeed, there is no second argument: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Messenger/Transport/Doctrine/DoctrineTransportFactory.php#L29